inc/azure_c_util/thandle.h

```diff
@@ -12,9 +12,11 @@
 
 #include "azure_c_logging/xlogging.h"
 
-#include "azure_c_util/containing_record.h"
 #include "azure_c_pal/refcount.h"
```

Comment:
>refcount [](start = 22, length = 8)

shouldn't this be interlocked? I thought THANDLE didn't use REFCOUNT anymore and just called interlocked directly...

Reply:
Of course it should :) Changed to interlocked.h

---
In reply to: [463688042](https://github.com/Azure/azure-c-util/pull/55#discussion_r463688042) [](ancestors = 463688042)